tests/chains/TheOpenNetwork/SignerTests.cpp

```diff
@@ -7,7 +7,7 @@
 #include "HexCoding.h"
 
 #include "TheOpenNetwork/Signer.h"
-#include "Everscale/Cell.h"
+#include "Everscale/CommonTON/Cell.h"
```

Comment:
As Everscale and TheOpenNetwork share common code, this can be in a standalone folder, preferably named `TON`, include directive will make it implicit that's common code shared between the 2 specific implementations.

Reply:
Hi, are you suggesting to move the folder to `src/TON` and use it from `src/Everscale` and `src/TheOpenNetwork`?  @catenocrypt mentioned that it is not right. 
>put common parts in a new folder on the level of chain folders — we do not have example of this, no-go"

Could you please agree on what is the best approach?

Reply:
Ok let's keep it like that for now.